### DISCUSSION TITLE
How to prevent specific css / js link in html to be resolved

### DISCUSSION THREAD

**Discussion by MaikoTan** [parcel-bundler/parcel#8503] [Q&A]
## Summary

Here is my `index.pug` like:

```pug
doctype html
html
  head
    link(href='/vendor/bootstrap.min.css' rel='stylesheet')
    script(src='/vendor/jquery.min.js')
    script(src='/vendor/bootstrap.min.js')
  body
    //- ...
```

I have `bootstrap` and `jquery` pre-bundled js / css already, and I can write a small script to copy those pre-bundled files to `dist/vendor/`.
How can I tell parcel could just ignore paths like `/vendor/bootstrap.min.css`, etc to be resolved and transformed.

**Comment by devongovett** [comment]
Why? Is something breaking when it's transformed?

**Reply by MaikoTan** [reply]
No, just I want to make it not to be tramsformed.
Like `http://some-cdn/bootstrap.min.css` would not be transformed, but there seems are not a possible way to mark a local path to not to be transformed.

**Reply by joshamaju** [reply]
Is it possible to achieve this? I have a similar situation but with iframes. I don't need parcel to resolve the src of the iframe

**Reply by mischnic** [reply]
See https://github.com/parcel-bundler/parcel/issues/1186, someone also built a plugin that may or may not work here: https://github.com/parcel-bundler/parcel/issues/1186#issuecomment-855249957
